pom.xml

```diff
@@ -886,6 +897,13 @@
           <relaxed>false</relaxed>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
This should be removed since a checkstyle enforcement is already part of the project.